source/server/http/admin.cc

```diff
@@ -762,9 +762,9 @@ Http::Code AdminImpl::handlerCerts(absl::string_view, Http::HeaderMap& response_
   server_.sslContextManager().iterateContexts([&](const Ssl::Context& context) -> void {
     envoy::admin::v2alpha::Certificate& certificate = *certificates.add_certificates();
     envoy::admin::v2alpha::CertificateDetails* ca_certificate = certificate.mutable_ca_cert();
-    ca_certificate->MergeFrom(*context.getCaCertInformation());
+    *ca_certificate = *context.getCaCertInformation();
```

Comment:
Doesn't this need to check for nullptr now? Same below? Can you add a test that covers this?